src/elements/common/media-query/useMedia.js

```diff
@@ -0,0 +1,101 @@
+// @flow
```

Comment:
It's not actually a component per-se, but this should live in the `components` directory rather than `elements`. Elements are top-level exports like ContentExplorer. The `common` folder within that is scoped to those use cases.